usr/libexec/greenboot/greenboot

```diff
@@ -7,6 +7,53 @@ SCRIPTS_CHECK_PATHS=("/usr/lib/greenboot/check" "/etc/greenboot/check")
 SCRIPTS_GREEN_PATHS=("/usr/lib/greenboot/green.d" "/etc/greenboot/green.d")
 SCRIPTS_RED_PATHS=("/usr/lib/greenboot/red.d" "/etc/greenboot/red.d")
 
+source_configuration_file() {
+    GREENBOOT_CONFIGURATION_FILE=/etc/greenboot/greenboot.conf
+    if test -f "$GREENBOOT_CONFIGURATION_FILE"; then
+        source $GREENBOOT_CONFIGURATION_FILE
+    fi
+}
+
+source_configuration_file
+function exists_in_list {
+    VALUE=$1
+    echo "${DISABLED_HEALTHCHECKS,,}" | tr " " '\n' | grep -F -q -x "$VALUE"
+}
+
+script_runner () {
+  local scripts_dir=$1; shift
+  local mode=$1; shift
+  local start_msg=$1; shift
+  local required_hc_failed=false
+  echo "$start_msg"
+  for script in $(find "$scripts_dir" -name '*.sh' | sort); do
+    if exists_in_list "$(basename "$script")"; then
```

Comment:
```suggestion
    if is_disabled "$(basename "$script")"; then
```

Reply:
Making this change for my next commit/force push, your name definitely makes a lot more sense than mine. I'll resolve this convo when I get that commit up